src/scss/variables/colors.scss

```diff
@@ -31,3 +31,9 @@ $error-color: #d64646;
 $success-color: #4bb678;
 $warning-color: #efa842;
 $info-color: #5ca8f5;
+$light-color: #cbd4de;
+
```

Comment:
we should discuss colors naming convention with @kamilmateusiak ;)

Reply:
Just removed the controversial color from variables, https://github.com/livechat/design-system/pull/85/commits/f883115cc7135279d240b0f31438e25bb244de80